script.js

```diff
@@ -22,3 +22,134 @@ function mobileMenu() {
 }
 
 mobileMenu();
+
+const cardPopUp = [
+  {
```

Comment:
- [ ] [OPTIONAL] I strongly recommend you to add padding on top of the modal window to match the design with figma, and make everything look even nicer:

Now  | After adding `margin-top`
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/146395213-4e5981dc-aa03-465f-940b-b55397f0e616.png) | ![image](https://user-images.githubusercontent.com/77038610/146395162-8af067d9-90d5-4302-8ae5-27394c2b6f8b.png)